stylesheets/cart.css

```diff
@@ -0,0 +1,126 @@
+.cart-main {
+  margin-top: 7.5rem;
+  scroll-margin-top: 7.5rem;
+  padding: 0 5rem;
+  display: flex;
+  gap: 6rem;
+  justify-content: center;
+}
+
+.cart-active {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
```

Comment:
this can be added to utility

Reply:
Plan to add this to my component library later. 👍